### PR TITLE
New portuguese (portugal) translation

### PR DESCRIPTION
**Pull request by jcfb-opensource**
translation in european portuguese for multimc

### CODE REVIEW

File: pt_PT.po [hunk 1]
```diff
@@ -0,0 +1,5500 @@
...
+#: src/application/dialogs/AboutDialog.ui:207
+msgctxt "AboutDialog|"
+msgid "Credits"
+msgstr "Créditos"
+
+#. Hey, Translator, feel free to put credit to you here
+#: src/application/dialogs/AboutDialog.ui:230
+msgctxt "AboutDialog|"
+msgid "No Language file loaded."
+msgstr "Ficheiro de língua não carregado."
```
Comment (RenanYudi) [inline pt_PT.po]: You can put something like "Versão em Português por: jcfb-opensource" here

**Comment by RenanYudi** [comment]
You also forgot to replace "arquivo" with "ficheiro"

**Comment by RenanYudi** [comment]
You also wrote "arquivo" many times, isn't it "ficheiro"?

**Comment by jcfb-opensource** [comment]
> 
> 
> You also wrote "arquivo" many times, shouldn't it be "ficheiro"?

Both mean the same, I wrote arquivo instead of ficheiro in some of them because it sounds better in european portuguese. I based my translation from the brazillian one with some hints from the template file. Maybe i'll improve the translation more and see if I can replace more words and make the translation more consistent

**Comment by RenanYudi** [comment]
I still think it should be "ficheiro", since Windows translates "file" as it.
And if "bifurcação" is a common word, it's better than "_Forking_/Redistribuição". I kept it as "forking" because no one here says that.

File: pt_PT.po [hunk 2]
```diff
@@ -0,0 +1,5500 @@
...
+
+#: src/application/pages/instance/LegacyUpgradePage.h:39
+msgctxt "LegacyUpgradePage|"
+msgid "Upgrade"
+msgstr "Atualizar"
+
+#: src/application/pages/instance/LegacyUpgradePage.ui:29
+msgctxt "LegacyUpgradePage|"
+msgid ""
+"<html><body><h1>Upgrade is requi
```
Comment (RenanYudi) [inline pt_PT.po]: WARN shouldn't be translated

Comment (RenanYudi) [inline pt_PT.po]: I think this string refers to the Minecraft Log or something related, so ERROR shouldn't be translated as the log output is in English.

Comment (RenanYudi) [inline pt_PT.po]: "Crash" ≠ "bloquear". "To crash" has no exact translation, the closest one in PT-BR is "travar", or simply, but incorrectly, "crashou". Minecraft PT-PT translates "crash" as "fechar", but that doesn't fit the context here.

Comment (RenanYudi) [inline pt_PT.po]: "Sobre" should be "Acerca"

File: pt_PT.po [hunk 3]
```diff
@@ -0,0 +1,5500 @@
...
+
+#: src/api/logic/minecraft/update/AssetUpdateTask.cpp:68
+msgctxt "AssetUpdateTask|"
+msgid "Failed to read the assets index!"
+msgstr "Falha ao ler o índice de ficheiros!"
+
+#: src/api/logic/minecraft/update/AssetUpdateTask.cpp:74
+msgctxt "AssetUpdateTask|"
+msgid "Getting the assets files from Mojang..."
+msgstr "A transferir ficheiros da Mojang..."
```
Comment (RenanYudi) [inline pt_PT.po]: "A transferir recursos…" or "A transferir ficheiros de recursos"

File: pt_PT.po (hunk 2 above)
Comment (RenanYudi) [inline pt_PT.po]: Untouched from old PT-BR. URL is a male noun.

File: pt_PT.po [hunk 4]
```diff
@@ -0,0 +1,5500 @@
...
+
+#: src/api/logic/minecraft/World.cpp:44
+msgctxt "GameType|"
+msgid "Adventure"
+msgstr "Aventura"
+
+#: src/api/logic/minecraft/World.cpp:46
+msgctxt "GameType|"
+msgid "Spectator"
+msgstr "Espectador"
```
Comment (RenanYudi) [inline pt_PT.po]: Minecraft translates as "Espetador"

File: pt_PT.po [hunk 5]
```diff
@@ -0,0 +1,5500 @@
...
+#: src/api/logic/updater/DownloadTask.cpp:136
+msgctxt "GoUpdate::DownloadTask|"
+msgid "Downloading one update file."
+msgstr "A transferir um arquivo de atualização."
+
+#: src/api/logic/updater/DownloadTask.cpp:140
+#, qt-format
+msgctxt "GoUpdate::DownloadTask|"
+msgid "Downloading %1 update files."
+msgstr "A transferir atualização... (%1 arquivos)"
```
Comment (RenanYudi) [inline pt_PT.po]: Inconsistent with the string above.
"A transferir %1 arquivos de atualização"

File: pt_PT.po (hunk 2 above)
Comment (RenanYudi) [inline pt_PT.po]: As far as I know "completidade" is not a word.

Comment (RenanYudi) [inline pt_PT.po]: Untouched from PT-BR. Double space "  ".

Comment (RenanYudi) [inline pt_PT.po]: Needs to be rewritten with "tu"

**Comment by peterix** [comment]
I'm moving translations over to a new porcess based on crowdin.

So, I'm going to merge this, as having it is better than not having it.

**Pull request merged**